### PR TITLE
Add Swift sample

### PR DESCRIPTION
**Pull request by jayjun**

### CODE REVIEW

**Comment by jonparrott** [comment]
This looks great, thanks for doing this.

@JustinBeckwith can I get your eyes on this real quick?

File: swift/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Use the latest Swift from Docker Hub
+FROM ibmcom/swift-ubuntu
+
+# Add current application
+ADD . .
```
Comment (JustinBeckwith) [inline swift/Dockerfile]: we typically add things to an /app folder in the container, may be nice to be consistent :)

File: swift/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Swift Custom Runtime for App Engine
+
+Simple sample of a [Swift](https://swift.org) app using [Kitura](http://www.kitura.io) that runs on [Google App Engine Flexible](https://cloud.google.com/appengine).
```
Comment (JustinBeckwith) [inline swift/README.md]: no need to say "flexible", just keep it "App Engine"

File: swift/Dockerfile (hunk 1 above)
Comment (jayjun) [inline swift/Dockerfile]: I'm just mirroring the other examples. Happy to change it, `.dockerignore` would be cleaner.

File: swift/README.md (hunk 2 above)
Comment (jayjun) [inline swift/README.md]: Likewise, I'm mirroring the README of other examples.

File: swift/Dockerfile (hunk 1 above)
Comment (jayjun) [inline swift/Dockerfile]: Sorry, my bad. All fixed!

**Pull request merged**